tests/Test.ADAL.NET.Common/Mocks/MockHelpers.cs

```diff
@@ -126,6 +127,26 @@ public static HttpResponseMessage CreateSuccessTokenResponseMessage()
             return CreateSuccessTokenResponseMessage(false);
         }
 
+        public static HttpResponseMessage CreateSuccessTokenResponseWithClientInfoMessage(string uniqueIdentifier, string unqiueTenantIdentifier)
```

Comment:
update existing functions to populate clientinfo. that way all existing tests get the new data. We can add separate tests for the cases when client_info is missing (it should be a failure case for AAD).